.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature request
+about: New feature for this project
+title: ''
+labels: ''
+assignees: ''
+---
+
+
+**User Story**
+> Add the details of this issue from the user's POV
+
+**Acceptance Criteria**
+> 1. **GIVEN** Given that something happens
+> **WHEN** Under certain conditions
+> **THEN** Then we expect a particular result
+
+**Tech Details**
+>Technical changes that will happen include UI changes, styling updates, integration requirements
+* **UI State Changes:**
+    >  Changes to the frontend interface
+* **Styling Updates:**
+    >  CSS changes for the application
+* **Integration Requirements:**
+    >  Changes to the logic parts of the application or protocol 
+
+
+**Design Link**
+>* Link to the figma design file 
```

Comment:
This should match paycrest/aggregator template. there's no UI changes to contracts